projects/ng2-qgrid-theme-material/src/lib/assets/tooltip.scss

```diff
@@ -0,0 +1,20 @@
+.q-grid-cell-tooltip {
+	position: fixed;
+	color: $white;
+	border-radius: 4px;
+	background: $markup-secondary-gray;
+	margin: 14px;
+	padding-left: 8px;
+	padding-right: 8px;
+	padding-top: 6px;
+	padding-bottom: 6px;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	width: fit-content;
+	max-width: 100%;
+	max-height: 100%;
+
+	&.q-grid-hide:not(.q-grid-show) {
```

Comment:
why you need .q-grid-show? =)

Reply:
Just saw similar logic in pager. But seems its not makes much sense here
removed